src/Compute/Compute/ChangeLog.md

```diff
@@ -29,7 +29,8 @@
 * Added `Zone` and `IntentVMSizeList` optional parameters to the cmdlet `New-AzProximityPlacementGroup`.
 
 ## Version 4.30.0
-* Added parameters `PackageFileName`, `ConfigFileName` for `New-AzGalleryApplicationVersion` 
+* Added parameters `PackageFileName`, `ConfigFileName` for `New-AzGalleryApplicationVersion`
+* Changes made to Gallery cmdlets for Community Galleries
```

Comment:
Please move the change log to the `Upcoming Release` section and make it past tense.